frontend/src/utils/url.ts

```diff
@@ -34,6 +34,12 @@ export const githubTournesolUrl = 'https://github.com/tournesol-app/tournesol';
 export const linkedInTournesolUrl =
   'https://www.linkedin.com/company/tournesol-app';
 
+// YouTube
+export const youtubePlaylistEnUrl =
+  'https://youtube.com/playlist?list=PLXX93NlSmUpayYeaNvKaeftX_QwEei3fU';
+export const youtubePlaylistBotFrUrl =
+  'https://youtube.com/playlist?list=PLXX93NlSmUpZkR9NaBXMaPlPalESb5tDv';
```

Comment:
Why is one called "En" and the other one "BotFr"? Is it a typo?

Reply:
oups, yes, I copy paste too fast